src/pages/default/MenuPageLayout.tsx

```diff
@@ -0,0 +1,81 @@
+import { Item, ItemProps } from "@/components/Item";
+import { ColorTheme } from "@/theme/colors";
+import { Layouts } from "@/theme/layouts";
+import { Typography } from "@/theme/typography";
+import React from "react";
+import {
+  SafeAreaView,
+  View,
+  ScrollView,
+  Pressable,
+  PressableProps,
+  Text,
+  StyleSheet,
+} from "react-native";
+
+export const MenuPageLayout = (props: {
+  headerItem: ItemProps;
+  menuTabs: MenuProps[];
+  children;
+}) => {
+  const { headerItem, menuTabs, children } = props;
+
+  return (
+    <SafeAreaView>
+      <View style={Layouts.container}>
+        <View style={Layouts.row}>
```

Comment:
I would rather work with flexbox directly than attempt to make it reusable like this